test/jdk/java/nio/file/FileStore/Basic.java

```diff
@@ -80,6 +80,16 @@ static void doTests(Path dir) throws IOException {
         assertTrue(store2.equals(store1));
         assertTrue(store1.hashCode() == store2.hashCode());
 
+        if (Platform.isWindows()) {
+            /**
+             * Test: FileStore.equals() should not be case sensitive
+             */
+            FileSystem fs = FileSystems.getDefault();
+            FileStore upper = Files.getFileStore(fs.getPath("C:\\"));
+            FileStore lower = Files.getFileStore(fs.getPath("c:\\"));
```

Comment:
Path.of("c:\\") can be used if you want.

Reply:
Will simplify to that in final commit.